src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -159,6 +159,14 @@ private static String convertElementNode(Node currentNode, String indentation, b
                     return generateStudentMotdPlaceholder(indentation);
                 }
             }
+        } else if (currentNode.getNodeName().equalsIgnoreCase("select")) {
```

Comment:
This is to prevent all the timezone options from being printed in the HTML test files, otherwise the LoCs here will reach ~1800 * number of test files affected (can easily reach 20000). This may or may not be temporary.